contract-tests/v3_api/conftest.py

```diff
@@ -1,4 +1,26 @@
-import urllib3
-
-# Disable any warnings about SSL connections
-urllib3.disable_warnings()
+import pytest
+import subprocess
+
+from faker import Faker
+
+
+@pytest.fixture(scope="session")
+def headers():
+    # Create a test user
+    fake = Faker()
+    email = fake.company_email()
+    user = fake.user_name()
```

Comment:
I think you can just drop the faker stuff out altogether and just use a hard coded test name like

`test-user@example.com`

Reply:
Done